.github/ISSUE_TEMPLATE/lgtm-com---false-positive.md

```diff
@@ -0,0 +1,18 @@
+---
+name: LGTM.com - false positive
+about: Tell us about an alert that shouldn't be reported
+title: LGTM.com - false positive
+labels: false-positive
+assignees: ''
+
+---
+
+**URL to the alert on the project page on LGTM.com**
+
+1. Open the project on LGTM.com.
+For example, https://lgtm.com/projects/g/pallets/click/.
+2. Switch to the `Alerts` tab. For example, https://lgtm.com/projects/g/pallets/click/alerts/.
+3. Scroll to the alert that you would like to report.
+4. Click on the right most icon `View this alert within the complete file`.
+5. A new browser tab opens. Copy and paste the page URL here.
+For example, https://lgtm.com/projects/g/pallets/click/snapshot/719fb7d8322b0767cdd1e5903ba3eb3233ba8dd5/files/click/_winconsole.py#xa08d213ab3289f87:1.
```

Comment:
these instructions should probably be in a comment to that they're not displayed by default

Reply:
Put `<!--` before and `-->` after the instructions, so that they are visible to the user doing the reporting, but not visible if they forget to delete it

Reply:
Ah you meant that, got it, fixed